source/manuals/security-overview-for-websites.html.md.erb

```diff
@@ -127,7 +126,9 @@ Enable relevant security headers to help your users' browsers understand what yo
 |Cross-Origin-Embedder-Policy|Sets the permission for how assets are loaded|
 |Cross-Origin-Opener-Policy|^|
 |Cross-Origin-Resource-Policy|^|
-|X-XSS-Protection|Enables older browsers' functionality to prevent XSS (cross-site scripting)|
+|Feature-Policy|*Partially deprecated* (see [OWASP guidance on Feature-Policy](https://owasp.org/www-project-secure-headers/#feature-policy)). Previously used to specify features the browser could access, primarily replaced with `Permissions-Policy`|
+|Expect-CT|*Deprecated* (see [OWASP guidance on Expect-CT](https://owasp.org/www-project-secure-headers/#expect-ct)). Previously used by browsers to check the certificate transparency logs, which helps determine the legitimacy of a HTTPS connection. It is no longer needed as a response header|
```

Comment:
Unfortunately the OWASP links don't work, for example https://owasp.org/www-project-secure-headers/#expect-ct takes you to the correct webpage but it doesn't show you the correct part of the page because it is hidden in a non active tab. You need to click on the 'Response headers' tab to see the information you need.

I don't know if you necessarily can/should do anything about this though.